code/pizzabot.js

```diff
@@ -1,7 +1,71 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
 
-//Put your Javscript code here:
+document.getElementById(
+  "welcome"
+).innerHTML = `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`;
+
+let orderName;
+let orderQuantity;
+
+// function to validate order. takes input from form and validates that pizza exist on the menu.
+// Returns true or false
+
+const validateOrderName = (validOrder) => {
+  return (
+    validOrder == vegetarian ||
+    validOrder == hawaiian ||
+    validOrder == pepperoni
+  );
+};
+
+// Function to calculate cooking time based on order quantity
+
+const calculateCookingTime = (orderQuantity) => {
+  if (orderQuantity >= 1 && orderQuantity <= 3) {
+    return 10;
+  } else if (orderQuantity < 6) {
+    return 15;
+  } else {
+    return 20;
+  }
+};
+
+// Function to calculate total cost based on order quantity
+
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+  return orderQuantity * pizzaPrice;
+};
+
+// Function to toggle active css class of order summary bubble (to show/hide)
+
+const toggle = () => {
+  return this.classList.toggle("active-summary");
```

Comment:
Noted. Reason for using this was I wasn't sure how the function would know what to change (since we´re not passing any arguments to the function). It works now when omitting both `return` and `this` but still a bit puzzled how it knows its making the assignment to the right element?

Reply:
I see. Happy to hear that you changed it at least :) Still, I don't want you to stay with doubts, so here is how I would implement this function :

This is function declaration
`const toggle = element => element.classList.toggle("active-summary");`

This is function call
`toggle(document.getElementById("orderSummary"))`

What do you think about that?